tests/vm_test.go

```diff
@@ -1267,3 +1272,40 @@ func createRunningVM(virtClient kubecli.KubevirtClient, template *v1.VirtualMach
 	Expect(err).ToNot(HaveOccurred())
 	return vm
 }
+
+func waitForVMStateTransition(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine, expectedStates []v1.VirtualMachinePrintableStatus, timeoutDuration time.Duration) {
```

Comment:
```suggestion
func waitForVMStateTransition(vm *v1.VirtualMachine, expectedStates []v1.VirtualMachinePrintableStatus, timeoutDuration time.Duration) {
```

Consider using `virtClient := kubevirt.Client()` instead.

Reply:
Applied the suggested change.